google-cloud-storage/acceptance/storage/file_test.rb

```diff
@@ -47,7 +47,7 @@
   end
 
   after do
-    bucket.files.all { |f| f.delete rescue nil }
+    bucket.files(versions: true).all { |f| f.delete generation: true rescue nil }
```

Comment:
Yes, I was unable to delete the bucket because there were versioned files that don't show when calling `bucket.files`, but do show when calling `bucket.files versions: true`. The acceptance helper and cleanup task were already updated to use this approach.